### ISSUE TITLE
Sharpness of text and icons

### ISSUE THREAD

**Issue by packet1** [joncage/ed-scout#153]
Hi,

In the PC app, the sharpness of images and icon is blurry compared to using the webserver and connecting with my ipad.  

PC screenshot:
![image](https://user-images.githubusercontent.com/70610088/103461832-d2566b00-4ce6-11eb-892b-41a26f940c8c.png)

Ipad screenshot:
![image](https://user-images.githubusercontent.com/70610088/103461847-e601d180-4ce6-11eb-8099-e95179ed3398.png)

**Comment by joncage** [comment]
Is that a retina iPad? I wonder if it's that the iPad is running at a higher resolution? The image there is 500 pixels wider I notice.

It's possible the PC direct interface using Chrome is picking up the icons differently Vs safari(?) running on the iPad

**Comment by packet1** [comment]
No, standard ipad8.   Probably the notable difference is the resolution of the displays.  PC is running @2k.  It's not jsut the icons, but the crispness of the text is different as well.

**Comment by joncage** [comment]
Is the PC running any special DPI settings?

**Comment by packet1** [comment]
![image](https://user-images.githubusercontent.com/70610088/103713189-5592ed80-4f81-11eb-86cf-6d90a7357783.png)
